universe/Condition.h

```diff
@@ -1964,6 +1964,30 @@ struct FO_COMMON_API Not final : public ConditionBase {
     void serialize(Archive& ar, const unsigned int version);
 };
 
+/** Matches all objects that match the first matching Condition in \a operands. */
```

Comment:
Maybe: Tests conditions in \a operands in order, to find the first condition that matches at least one candidate object. Matches candidate objects that match that condition, ignoring any conditions listed later. If no candidate matches any of the conditions, returns nothing.